examples/templates/README.md

```diff
@@ -1,21 +1,45 @@
-## Templates Guidline
+## Templates guideline
 
-These templates are a base for the new code examples.
+This folder contains templates for new code examples. Each new code example should be created using the template from this folder. We have templates for examples made in React, Angular, Vue, and of course Vanilla JavaScript.
+
+#### Prerequisites
+The lowest version of Node.js required for working with this project is `15.9.0`. If you use `nvm` (Node Version Manager) and already installed the minimum supported version, just type the following command in the terminal: `nvm use`.
 
 ### Creating a new example
 
-All new code examples should be added to the `/examples/next` directory. Based on them you can create versioned code examples.
+In this guideline, we will walk you through the process of creating new code examples. 
+
+> If you want to get inspired before moving forward, see the already existing code examples in the folder `examples/next/docs/`.
+
+First off, create a new branch in order to keep the development process clean.
+
+Inside the `examples/templates` directory where all templates for future code examples live. You can see that we have templates for examples made in React, Angular, Vue, and of course vanilla JavaScript.
```

Comment:
We use 'Vanilla' capitalized everywhere else, so we should capitalize this one as well.